node/src/daemon/daemon_initializer.rs

```diff
@@ -162,15 +155,33 @@ impl DaemonInitializerReal {
             .collect_vec();
         self.rerunner.rerun(param_vec);
     }
+
+    #[cfg(test)]
```

Comment:
I cannot; it's unusual but this function is needed in a test in a different file where the test can be more complex. Still I have to create the method here because I declared the struct here.